process/coordinator/process.go

```diff
@@ -719,6 +719,11 @@ func (tc *transactionCoordinator) CreateMbsAndProcessCrossShardTransactionsDstMe
 				"total gas penalized", tc.gasHandler.TotalGasPenalized(),
 				"error", errProc,
 			)
+
+			if core.IsGetNodeFromDBError(errProc) {
+				return nil, 0, false, err
```

Comment:
The intention is that if a certain tx will find a missing trie node, that validator should not propose/sign blocks until that node is synced. This will help in case a certain trie node is missing from all validators in order to not have consensus on trie not found err.